tests/test_config.py

```
@@ -1498,5 +1490,151 @@ def test_apply_env_config_overrides_matrix_password_override(self):
         self.assertEqual(result["matrix"]["bot_user_id"], "@bot:example.org")
 
 
+class TestConfigUncoveredLines(unittest.TestCase):
+    """Test uncovered lines in config.py."""
+
+    @patch("mmrelay.config.os.path.exists")
+    @patch("mmrelay.config.get_base_dir", return_value="/test/base")
+    def test_load_credentials_no_file_windows(self, mock_get_base_dir, mock_exists):
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_get_base_dir`

(ARG002)